sql/internal/specutil/spec.go

```diff
@@ -0,0 +1,45 @@
+package specutil
+
+import (
+	"strconv"
+
+	"ariga.io/atlas/schema/schemaspec"
+	"ariga.io/atlas/sql/sqlspec"
+)
+
+// NewCol is a helper method for constructing *sqlspec.Column instances.
+func NewCol(name, coltype string, attrs ...*schemaspec.Attr) *sqlspec.Column {
```

Comment:
```suggestion
func Col(name, coltype string, attrs ...*schemaspec.Attr) *sqlspec.Column {
```

Why only this function has a `New` prefix?